### PR TITLE
DOC-2578: Pressing `Shift + Enter` in the comment textarea now selects the highlighted user.

### PR DESCRIPTION
**Pull request by abhinavgandham**
Ticket: DOC-2578

Site: [Staging branch](http://docs-feature-760-doc-2578tiny-11455.staging.tiny.cloud/docs/tinymce/latest/7.6.0-release-notes/#pressing-shiftenter-in-mentions-in-comments-dropdown-should-accept-the-active-menu-item)

Changes:
* Documented the improvement for TINY-11455.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.6.0-release-notes.adoc
Comment (kemister85) [inline modules/ROOT/pages/7.6.0-release-notes.adoc:117]:
```suggestion
In previous versions of {productname}, the `Shift+Enter` key was not properly handled when used with mentions in comments. This caused a new line to be inserted while the mentions dropdown remained open, leading to inconsistent behavior compared to mentions within the editor.

In {productname} {release-version}, mentions in comments now handle the `Shift+Enter` key correctly by inserting the highlighted user into the comment text area and closing the dropdown. This ensures consistent functionality across the editor and comment areas.
```

Comment (kemister85) [reply modules/ROOT/pages/7.6.0-release-notes.adoc:117]: Feedback: when referring to TinyMCE replace each instance with `{productname}`, and the same for the current `{release-version}`.

**Review by kemister85 — APPROVED** [review]

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/7.6.0-release-notes.adoc
Comment (kemister85) [inline modules/ROOT/pages/7.6.0-release-notes.adoc:117]: This needs to be moved under `Accompanying Premium plugin changes` for Comments plugin.

```
=== Comments

The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.

**Comments** Premium plugin includes the following additions and improvement.

==== Title
// #TINY-xxxx

<Insert Here>

For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Introduction to {companyname} Comments].

```

Comment (abhinavgandham) [reply modules/ROOT/pages/7.6.0-release-notes.adoc:117]: @FarzadHayat Noted, so I will wait until your section is merged first, and then I will push my changes.

File: modules/ROOT/pages/7.6.0-release-notes.adoc
Comment (kemister85) [inline modules/ROOT/pages/7.6.0-release-notes.adoc:62]:
```suggestion
The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.
```

**Pull request merged**
